xdsl/dialects/riscv.py

```diff
@@ -1829,6 +1832,43 @@ def assembly_line(self) -> str | None:
         return _assembly_line(self.directive.data, (value,), is_indented=False)
 
 
+@irdl_op_definition
+class CustomInstructionOp(IRDLOperation, RISCVInstruction):
```

Comment:
A docstring would be super nice here, as it is quite non-standard (and even somewhat meta).